smokerd.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=GoodData Smoke testing daemon
+
+[Service]
+Type=simple
+
+Environment=CONFIG=/etc/smokerd/smokerd.yaml
+Environment='OPTIONS=-v --foreground --no-console'
```

Comment:
Because the best way how to use systemd is to keep services running on foreground. Using ```Type=forking``` is always considered as a last resort, because systemd cannot properly manage daemonized processes.